packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -475,8 +476,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
             onClick={ () => this._onItemClick(item.index!) }
             role='option'
             aria-selected={ this.state.selectedIndex === item.index ? 'true' : 'false' }
-            ariaLabel={ item.ariaLabel || item.text }
-            title={ item.text }
```

Comment:
if there is no ariaLabel should we fallback to item.text?

Reply:
@betrue-final-final the concern here is that ariaLabel and title are redundant. Does the dropdown need both if ariaLabel and text are different, or should we only use one....title OR ariaLabel